src/actions/profile.js

```diff
@@ -415,6 +423,7 @@ function updatePasswordDone(profile, tokenV3, newPassword, oldPassword) {
 export default createActions({
   PROFILE: {
     LOAD_PROFILE: loadProfile,
+    CLEAR_PROFILE: clearProfile,
```

Comment:
Could we just use Lodash's `noop` instead of an empty method?

Reply:
@suppermancool

Reply:
let me try

Reply:
@ThomasKranitsas you mean _.noop ?

Reply:
yes

Reply:
yeah noop work, updated the code, pls check

Reply:
@ThomasKranitsas